text/0000-custom-test-frameworks.md

```diff
@@ -173,9 +167,7 @@ This section works like this:
 
 ```rust
 [testing.framework]
-folders = ["bench"]
-lib = true              # true by default
-single-target = true    # false by default
+kind = "test" # or bench
```

Comment:
Do we want to require that a framework is `test` xor `bench`?

Reply:
Yes, we're not allowing for new folders or new kinds right now.

In the future, we can allow for more varieties here.